### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by belzy**

### CODE REVIEW

File: api/models.py [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+    username   = models.CharField(max_length=20)
+    password   = models.CharField(max_length=20)
+    premium    = models.BooleanField()
+    created_at = models.DateTimeField()
+
+class Subtopic(models.Model):
+    name       = models.CharField(max_length=20)
+    access     = models.BooleanField()
+    created_at = models.DateTimeField()
+    owner_id   = models.IntegerField()
```
Comment (xpuentes) [inline api/models.py:15]: owner_id should be a Foreign Key
owner_id = models.ForeignKey(User)

File: api/models.py [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+
+class Discussion(models.Model):
+    title       = models.CharField(max_length=20)
+    description = models.TextField(max_length=200)
+    created_at  = models.DateTimeField()
+    owner_id    = models.IntegerField()
```
Comment (xpuentes) [inline api/models.py:21]: owner_id should be a Foreign Key
owner_id = models.ForeignKey(User)

File: api/models.py [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+    subtopic_id = models.IntegerField()
```
Comment (xpuentes) [inline api/models.py:22]: suptopic_id should be a Foreign Key
subtopic_id = models.ForeignKey(Subtopic)

File: api/models.py [hunk 4]
```diff
@@ -1,3 +1,33 @@
...
+
+class Comments(models.Model):
+    text          = models.TextField(max_length=200)
+    created_at    = models.DateTimeField()
+    owner_id      = models.IntegerField()
```
Comment (xpuentes) [inline api/models.py:27]: owner_id should be a Foreign Key
owner_id = models.ForeignKey(User)

File: api/models.py [hunk 5]
```diff
@@ -1,3 +1,33 @@
...
+    discussion_id = models.IntegerField()
```
Comment (xpuentes) [inline api/models.py:28]: discussion_id should be a Foreign Key
discussion_id = models.ForeignKey(Discussion)

File: api/models.py [hunk 6]
```diff
@@ -1,3 +1,33 @@
...
+
+# Relationships
+class UserToSubtopic(models.Model):
+    user_id     = models.IntegerField()
```
Comment (xpuentes) [inline api/models.py:32]: owner_id should be a Foreign Key
owner_id = models.ForeignKey(User)

File: api/models.py [hunk 7]
```diff
@@ -1,3 +1,33 @@
...
+    subtopic_id = models.IntegerField()
```
Comment (xpuentes) [inline api/models.py:33]: suptopic_id should be a Foreign Key
subtopic_id = models.ForeignKey(Subtopic)

**Review by xpuentes — COMMENTED** [review]
Update fields to Foreign Keys

**Review by xpuentes — APPROVED** [review]

**Review by abravebee — APPROVED** [review]

**Review by evan-leduc — APPROVED** [review]

**Pull request merged**